compiler/one-cmds/one-init

```diff
@@ -275,6 +365,11 @@ def main():
     # verify arguments
     _verify_arg(parser, args)
 
+    #TODO generate cfg file
```

Comment:
AFAIU, `_generate()` writes cfg file. What does `TODO` in the comment mean?

Reply:
Thanks. I removed!